nemo_curator/utils/fuzzy_dedup_utils/merge_utils.py

```diff
@@ -195,6 +212,10 @@ def merge_left_to_shuffled_right(
     subset_bucket_df,
     merge_on,
 ):
+    from nemo_curator.utils.fuzzy_dedup_utils.shuffle_utils import (
+        rearange_by_column_direct,
+    )
```

Comment:
Note that I moved this import into `merge_left_to_shuffled_right`, because `shuffle_utils` currently requires `cudf`/`dask_cuda`, while other logic in this module does not.